app/models/amr_data_feed_config.rb

```diff
@@ -59,4 +65,13 @@ def header_first_thing
   def mpan_mprn_index
     map_of_fields_to_indexes[:mpan_mprn_index]
   end
+
+  def s3_archive_folder
+    "archive-#{identifier}"
+  end
+
+  def local_bucket_path
+    path = ENV['AMR_CONFIG_LOCAL_FILE_BUCKET_PATH'] || 'tmp/amr_files_bucket'
+    "#{path}/#{identifier}"
+  end
```

Comment:
These replace the database columns - with added override if required for the local bucket path